src/4.3_to_5.0/__test_resources__/emptyLegacyUIFolder.ts

```diff
@@ -0,0 +1,214 @@
  "columnName": "pnlDelta"\n          },\n          {\n            "functionName": "Caption",\n            "columnName": "pnlDelta"\n          },\n          {\n            "functionName": "MemberValue",\n            "columnName": "pnl"\n          },\n          {\n            "functionName": "Caption",\n            "columnName": "pnl"\n          },\n          {\n            "functionName": "MemberValue",\n            "columnName": "gamma"\n          },\n          {\n            "functionName": "Caption",\n            "columnName": "gamma"\n          },\n          {\n            "functionName": "MemberValue",\n            "columnName": "delta"\n          },\n          {\n            "functionName": "Caption",\n            "columnName": "delta"\n          },\n          {\n            "functionName": "MemberValue",\n            "columnName": "TradeId"\n          },\n          {\n            "functionName": "Caption",\n            "columnName": "TradeId"\n          },\n          {\n            "functionName": "MemberValue",\n            "columnName": "ProductQtyMultiplier"\n          },\n          {\n            "functionName": "Caption",\n            "columnName": "ProductQtyMultiplier"\n          },\n          {\n            "functionName": "MemberValue",\n            "columnName": "ProductQtyMultiplier"\n          },\n          {\n            "functionName": "Caption",\n            "columnName": "ProductQtyMultiplier"\n          },\n          {\n            "functionName": "MemberValue",\n            "columnName": "ProductBaseMtm"\n          },\n          {\n            "functionName": "Caption",\n            "columnName": "ProductBaseMtm"\n          },\n          {\n            "columnName": "ProductQtyMultiplier",\n            "functionName": "MemberValue"\n          },\n          {\n            "columnName": "ProductQtyMultiplier",\n            "functionName": "Caption"\n          }\n        ]\n      }\n    },\n    "defaultPermissions": {\n      "owners": [\n        "admin"\n      ],\n      "readers": [\n        "admin"\n      ]\n    },\n    "project.content-editor.configuration": {\n      "autoSwitchToFieldsOnEmptyWidget": true,\n      "advancedModeDropdownHidden": false,\n      "hideFieldsControls": false,\n      "mdx-common": {\n        "regularModeTab": "filters",\n        "advancedModeEnabled": false,\n        "advancedModeTab": "mdx"\n      }\n    },\n    "user.contextValues": {\n      "EquityDerivativesCube": {},\n      "EquityDerivativesCubeDist": {}\n    },\n    "user.filters": {\n      "EquityDerivativesCube": ["[Geography].[City].[ALL].[AllMember].[Berlin]"],\n      "EquityDerivativesCubeDist": []\n    },\n    "filtering.hierarchyToFilterType.[Geography].[City]": "explicit",\n    "filtering.hierarchyToFilterType.[Currency].[Currency]": "explicit",\n    "filtering.hierarchyToFilterType.[Trades].[Trades]": "topcount",\n    "servers.alias": {\n      "https://activepivot-ranch.activeviam.com:5900": "http://localhost:8080"\n    }\n  }\n}',
```

Comment:
I would prefer to not have this in `emptyLegacyUIFolder` and to have it actually be empty.

Reply:
@Nouzbe Removing this would mean losing the `settings` in the snapshot for the `migrate_43_to_50` test since now we're generating `smallLegacyUIFolder` with this file and injecting the bookmark content via the `addLegacyBookmarkToUIFolder`. I'll keep this as is.